odxtools/unitgroup.py

```diff
@@ -11,7 +12,10 @@
 if TYPE_CHECKING:
     from .diaglayer import DiagLayer
 
-UnitGroupCategory = Literal["COUNTRY", "EQUIV-UNITS"]
+
+class UnitGroupCategory(Enum):
+    COUNTRY = "COUNTRY"
+    EQUIV_UNITS = "EQUIV-UNITS"
```

Comment:
Are you sure Category only accepts those values?

Reply:
I just checked. it is weird IMO, but true:
![image](https://github.com/mercedes-benz/odxtools/assets/1837735/f9004da5-50bf-46cb-9fc5-1a1093d10d61)